pkg/runner/runner.go

```diff
@@ -2,27 +2,61 @@ package runner
 
 import (
 	"errors"
+	"fmt"
 	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/kelseyhightower/envconfig"
 	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
 	"github.com/kubeshop/testkube/pkg/executor/content"
 	"github.com/kubeshop/testkube/pkg/executor/output"
+	"github.com/kubeshop/testkube/pkg/executor/scrapper"
 )
 
+type Params struct {
+	Endpoint        string // RUNNER_ENDPOINT
+	AccessKeyID     string // RUNNER_ACCESSKEYID
+	SecretAccessKey string // RUNNER_SECRETACCESSKEY
+	Location        string // RUNNER_LOCATION
+	Token           string // RUNNER_TOKEN
+	Ssl             bool   // RUNNER_SSL
+}
+
 // NewRunner creates a new SoapUIRunner
 func NewRunner() *SoapUIRunner {
+	var params Params
+	err := envconfig.Process("runner", &params)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Panicing doesn't look like a good idea. I would log error and exit

Reply:
I will update this, good catch

Reply:
moved the os.Exit() to the main, as it is a better to break the flow this way as high as possible